common/djangoapps/student/tests/tests.py

```diff
@@ -1145,3 +1169,133 @@ def _assert_responses(self, response, count):
         self.assertContains(response, 'This course is 1 of 3 courses in the', count)
         self.assertContains(response, self.program_name, count)
         self.assertContains(response, 'View XSeries Details', count)
+
+    def test_get_programs_method_without_cache(self):
```

Comment:
```python
def test_get_programs_without_cache(self):
""" Verify that if the programs config has cache disabled than
the method '_get_course_programs' will not cache the response
and it will hit the Programs API for each request.
```